apps/voting/app/src/components/VotePanelContent.js

```diff
@@ -390,4 +385,8 @@ const VotingButton = styled(Button)`
   }
 `
 
-export default provideNetwork(VotePanelContent)
+export default props => (
+  <NetworkContext.Consumer>
```

Comment:
Is it too big a change to refactor from a `class` to `function` Component?

Reply:
Yes I didn’t want to touch it for now as it is quite self contained, so we can focus on `App.js` in this PR.